client/src/pages/Home.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import Headlines from "../components/page-components/Headlines"
+import StoryCard from "../components/page-components/StoryCard";
+import { Link } from "react-router-dom"
+import TopicSection from "../components/page-components/TopicSection";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+type Topic = {
+    topic: string,
+    color: string
+  };
+
+type Props = {
+    topics: Topic[]
+}
+
+type Stories = {
+    stories: Story[]
+}
+
+type Story = {
+    name: string,
+    author: string,
+    body: string,
+    source_url: string,
+    cover_url: string,
+    category: string
+}
+
+const Home = (props: Props) => {
+
+    const [data, setData] = useState<any>();
+        
+        const getData = async () => {
+            try {
+                const response = await axios.get("http://localhost:3002/article.list");
```

Comment:
Can we store the domain for the server in an environment variable. Vite should automatically load in variables from `.env` files if the variable is prefixed with `VITE_`

https://vitejs.dev/guide/env-and-mode.html#env-files

```env
# client/.env
VITE_SERVER_DOMAIN = http://localhost:3001/ # or 3002 😉
```

```typescript
// someReactFile.tsx
const domain = import.meta.VITE_SERVER_DOMAIN
```

Reply:
Good idea, have updated to include this